src/main/webapp/country.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @param {query} id
+ */
+async function searching(id) {
+  const countryResponse = await fetch('/api/countries/' + id);
+  const country = await countryResponse.json();
+  const citiesResponse = await fetch('/api/cities/');
+  const cities = await citiesResponse.json();
+
+  const citiesInCountry = [];
+  for (const city in cities) {
+    if (city.countryID === id) {
+      citiesInCountry.push(city);
+    }
+  }
+
```

Comment:
I know this wasn't documented in the API docs, but you can actually specify a [`countryId`](https://github.com/gferioli0418/travelbud/blob/master/src/main/java/com/google/sps/travelbud/CityServlet.java#L41) query parameter that does the filtering for you. So this can be:
```suggestion
  const citiesResponse = await fetch('/api/cities?countryId=' + country.id);
  const citiesInCountry = await citiesResponse.json();
```

Reply:
FYI, I merged the PR to get the changes in, but if you like this suggestion or parts of it please feel free to address in a new PR